cmd/_integration-tests/transport/http_test.go

```diff
@@ -239,6 +239,29 @@ func TestCtxToCtxViaHTTPHeaderRequest(t *testing.T) {
 	}
 }
 
+// Test that making a get request with capital letters in the path functions
+// correctly.
+func TestGetWithCapsPath(t *testing.T) {
```

Comment:
Could you add a test for a standard HTTP request as well? 

See [func TestGetWithQueryRequest](https://github.com/hasLeland/go-truss/blob/2bc8b2855686f329b05aa15675eb3026e2d0909c/cmd/_integration-tests/transport/http_test.go#L45) for an example